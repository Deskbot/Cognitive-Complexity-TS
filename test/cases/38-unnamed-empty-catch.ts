try {
  console;
} catch {
  // If the session delete fails then it is likely that the session has
  // already expired anyway so do nothing.
}
