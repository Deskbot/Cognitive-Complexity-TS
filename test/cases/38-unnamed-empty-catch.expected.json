{
    "38-unnamed-empty-catch.ts": {
        "score": 1
    }
}